lib/resty/dns/resolver.lua

```diff
@@ -846,56 +846,61 @@ function _M.query(self, qname, opts)
     -- local cjson = require "cjson"
     -- print("query: ", cjson.encode(concat(query, "")))
 
+    local ok
     local retrans = self.retrans
+    if tries then
+      tries[1] = nil
```

Comment:
2 space indentation? We use 4-space indentation consistently in this library. Please fix other similar places as well (I noted another such place at least).